src/assessment.js

```diff
@@ -19,16 +19,16 @@
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
-//
+var q1 = false;
+// false
 
 // Which scope(s) may access the variable `laswerQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
-//
+var q2 = [fire, deathStar];
```

Comment:
`deathStar` is an object, not a function; as such, it doesn't have its own scope. Since `deathStar` is in the `global` scope, so are all of its properties, including `laserQuantity`. And since the `fire` scope is inside the `global` scope, it can access anything that's available in `global`. The correct answer to this question is `['fire', 'global']`